core_api/src/dependencies.py

```diff
@@ -35,9 +34,14 @@ def get_elasticsearch_client(env: Annotated[Settings, Depends(get_env)]) -> Elas
 
 @lru_cache(1)
 def get_embedding_model(env: Annotated[Settings, Depends(get_env)]) -> Embeddings:
-    embedding_model = SentenceTransformerEmbeddings(model_name=env.embedding_model, cache_folder=MODEL_PATH)
-    log.info("Loaded embedding model from environment: %s", env.embedding_model)
-    return embedding_model
+    return AzureOpenAIEmbeddings(
+        azure_endpoint=env.azure_openai_endpoint,
+        openai_api_version="2023-05-15",
```

Comment:
Should this be using env.openai_api_version instead of hardcoded? It mismatches our current default version, so we might want to consider updating to the version you note here.